backend/api/views.py

```diff
@@ -20,6 +21,62 @@ def api_teambuilder():
     return get_team(archetype)
 
 
+@views.route("battle", methods=["GET"])
```

Comment:
Maybe we will have to define a more specific naming for the API. We can see that later when we design the final API.